week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,4 +1,9 @@
+function add (num1, num2) {
+    return num1 + num2;
+
+}
 // Declare your function first
+var sum  = (13 + 124);
```

Comment:
Your function above is correct! Please use it here too. Also, the parenthesis here are not necessary 👍